addon/mixins/pikaday.js

```diff
@@ -136,14 +137,19 @@ export default Mixin.create({
   },
 
   setMinDate: function() {
-    const { pikaday, minDate, value } = getProperties(this, [
+    let { pikaday, minDate, value } = getProperties(this, [
       'pikaday',
       'minDate',
       'value'
     ]);
 
     if (minDate) {
-      const _minDate = new Date(minDate.getTime());
+      try {
+        minDate = minDate.getTime();
+      } catch (e) {
+        minDate = new Date(minDate);
+      }
+      const _minDate = new Date(minDate);
```

Comment:
This seems a little suspect to me... Why `new Date(...)` a value that potentially has already been turned into a `new Date(...)?

If we need to convert `minDate` to a `Date` object within the `try` condition, I think it would make more sense to do that there than pass `minDate` to the `Date` constructor twice.